notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -551,7 +551,7 @@
     "xtalk_overlaps = {}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def construct_filter(times, fc, fhw, eigval_cutoff=FR_EIGENVAL_CUTOFF, wgts=None):
</pre><p>Let's add a TODO note that this function should be graduated into hera_cal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>